src/main/scala/org/constellation/trust/EigenTrust.scala

```diff
@@ -23,14 +26,14 @@ object EigenTrust {
   nodesWithEdges.foreach { node: TrustNode =>
     node.edges.foreach { edge =>
       val trust = edge.trust / 2 + 0.5 // Revert from -1 to 1 => 0 to 1
-//      println(trust)
+      //      println(trust)
```

Comment:
probably should be removed

Reply:
Most of the code inside EigenTrust will be probably removed but for now let's leave it. We will clean that object anyway after switching the logic